core/arch/arm/include/ffa.h

```diff
@@ -0,0 +1,77 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright (c) 2020, Linaro Limited
+ * Copyright (c) 2018-2019, Arm Limited. All rights reserved.
+ */
+
+#ifndef __FFA_H
+#define __FFA_H
+
+/* This is based on the FF-A 1.0 EAC specification */
+
+#include <smccc.h>
+#include <stdint.h>
+
+/* Error codes */
+#define FFA_NOT_SUPPORTED	-1
+#define FFA_INVALID_PARAMETERS	-2
+#define FFA_NO_MEMORY		-3
+#define FFA_BUSY		-4
+#define FFA_INTERRUPTED		-5
+#define FFA_DENIED		-6
+#define FFA_RETRY		-7
```

Comment:
There's also an ABORTED (-8) error code in the specification.

Reply:
Thanks, I'll add it.